pkg/provider/fixtures/annotations-traffic-type-topology.json

```diff
@@ -20,6 +20,27 @@
         "pod-a1@my-ns",
         "pod-a2@my-ns"
       ]
+    },
+    "svc-b@my-ns": {
+      "name": "svc-b",
+      "namespace": "my-ns",
+      "selector": {},
+      "annotations": {
+        "maesh.containo.us/traffic-type": "udp"
```

Comment:
Is this annotation required? since the port is specifically UDP?

Reply:
If the `maesh.containo.us/traffic-type` annotation is not defined, Maesh uses the configured default traffic type mode. So, if the default mode is `HTTP` the `UDP` ports will not be available through Maesh.